getting-started/support_matrix.md

```diff
@@ -19,6 +19,23 @@ KubeArmor supports following types of workloads:
 | VMWare Tanzu | * | TBD |
 | Nutanix | * | TBD |
 
+## Supported Linux Distributions
+
+| Provider | Distro | VM / Bare-metal | Kubernetes |
+|----------|--------|---------------|------|
+| SUSE | SUSE Enterprise 15 | Full | Full |
+| Debian | [Buster](https://www.debian.org/releases/buster/) / [Bullseye](https://www.debian.org/releases/bullseye/) | Full | Full |
+| Ubuntu | 18.04 / 20.04 | Full | Full |
+| RedHat / CentOS | RHEL 8.4 / CentOS 8.4 | Full | Partial |
```

Comment:
I am not sure if we should mention Full support at all in case of SELinux only systems.

Reply:
I have tried it on RHEL 8.5 with BPF LSM disabled and enforcement is running fine with SELinux on Host based policy.

```
== Alert / 2022-11-29 18:39:07.163303 ==
HostName: ankur-test
Type: MatchedHostPolicy
PolicyName: hsp-kubearmor-dev-file-path-block
Severity: 5
Source: /usr/bin/cat /etc/hostname
Resource: /etc/hostname
Operation: File
Action: Block
Data: syscall=SYS_OPENAT fd=-100 flags=O_RDONLY
Enforcer: SELinux
Result: Permission denied
HostPID: 3627
HostPPID: 2895
PID: 3627
PPID: 2895
ParentProcessName: /usr/bin/bash
ProcessName: /usr/bin/cat
UID: 1000
```

Reply:
![image](https://user-images.githubusercontent.com/39901181/204618116-3a63cba0-14fe-42cd-9681-caf33819b6ae.png)